tst/EnergyPlus/unit/ChillerElectricEIR.unit.cc

```diff
@@ -371,3 +371,166 @@ TEST_F(EnergyPlusFixture, ChillerElectricEIR_EvaporativelyCooled_Calculate)
     EXPECT_NEAR(6.22019725E-06, EvapCondWaterVolFlowRate, 0.000000001);
     EXPECT_NEAR(EvapCondWaterVolFlowRate, thisEIRChiller.EvapWaterConsumpRate, 0.000000001);
 }
+
+TEST_F(EnergyPlusFixture, ChillerElectricEIR_WaterCooledChillerVariableSpeedCondenser)
```

Comment:
New tests for both chiller models that tests all three new control option as well as the minimum condenser flow fraction.